qiskit_ibm_runtime/fake_provider/fake_backend.py

```diff
@@ -165,6 +174,101 @@ def _load_json(self, filename: str) -> dict:
             the_json = json.load(f_json)
         return the_json
 
+    @property
+    def service(self) -> QiskitRuntimeLocalService:
+        """Return the ``service`` object
+
+        Returns:
+            service: instance of QiskitRuntimeLocalService
+        """
+        return self._service
+
+    def status(self) -> BackendStatus:
+        """Return the backend status.
+
+        Returns:
+            The status of the backend.
+
+        """
+
+        api_status = {
+            "backend_name": self.name,
+            "backend_version": "",
+            "status_msg": "active",
+            "operational": True,
+            "pending_jobs": 0,
+        }
+
+        return BackendStatus.from_dict(api_status)
+
+    def properties(self, refresh: bool = False) -> BackendProperties:
+        """Return the backend properties
+
+        Args:
+            refresh: If ``True``, re-retrieve the backend properties
+            from the local file.
+
+        Returns:
+            The backend properties.
+        """
+        if refresh or (self._props_dict is None):
+            self._set_props_dict_from_json()
+        return BackendProperties.from_dict(self._props_dict)
+
+    def defaults(self, refresh: bool = False) -> PulseDefaults:
```

Comment:
The parameter name could be confusing because we have a `refresh` method to update the data files from their real. Maybe `reload` sounds better?

Reply:
I think we should be consistent with the methods in `ibm_backend` - also it's the same refresh because we want to use the files that were just updated by the `refresh()` method